Update overwrite config files and topology makelist for ADL-based released products

johnylin76 opened the pull request:
Commits in this PR provide binary/topology build config update for released products (ADL-based). The update includes:

- overwrite config files for firmware binary builds
- topology make entries and topology modification
- memory map resizing under specific configs

johnylin76 commented:
@MingJenTai for viz

https://github.com/thesofproject/sof/pull/6612/commits/c04ce1a386fa7a1da4e2cf187b7b6fd4e0c75ca8 in this PR unifies DMICPROC `rtnr-google-rtc-processing` and `google-rtc-processing-rtnr`. They are the same graph actually.

marc-hb commented:
Can you provide a pull request title and description that describes the changes? Where they come from is interesting information but it's really secondary.

Maybe split this in several pull requests if some commits are unrelated to the others.

johnylin76 commented:
> Can you provide a pull request title and description that describes the changes? Where they come from is interesting information but it's really secondary.
> 
> Maybe split this in several pull requests if some commits are unrelated to the others.

Thanks for the suggestion.
Just updated PR title and description, and fixed checkpatch error.
